ros/src/computing/perception/detection/lidar_detector/packages/lidar_fake_perception/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(lidar_fake_perception)
+
+add_compile_options(-std=c++11)
```

Comment:
can be removed because autoware_build_flags sets -std=c++11 at #1395.

Reply:
fixed, thanks. btw, do you have plan about this common build option?

Reply:
@aohsato It'd be better to add some options for controlling the entire quality of Autoware gradually.
-Werror is a candidate but it should be introduced carefully.

Reply:
@kfunaoka I see. In future, please add rules about using this flags to our documentation. :)

Reply:
@aohsato Of cousrce 👍 
https://github.com/CPFL/Autoware/wiki/Quality-Control